docs/data-sources/password_settings.md

```diff
@@ -3,12 +3,12 @@
 page_title: "oktapam_password_settings Data Source - terraform-provider-oktapam"
 subcategory: ""
 description: |-
-  Returns an existing Password Policy for a PAM Project.  For details, see Projects https://help.okta.com/en/programs/opa-pam/Content/Topics/privileged-access/pam-projects.htm
+  Returns an existing Password Policy for a PAM Project.  For details, see Projects https://help.okta.com/okta_help.htm?type=oie&id=ext-pam-projects.
 ---
 
 # oktapam_password_settings (Data Source)
 
-Returns an existing Password Policy for a PAM Project.  For details, see [Projects](https://help.okta.com/en/programs/opa-pam/Content/Topics/privileged-access/pam-projects.htm)
+Returns an existing Password Policy for a PAM Project.  For details, see [Projects](https://help.okta.com/okta_help.htm?type=oie&id=ext-pam-projects).
```

Comment:
This link takes me to the generic OIE docs: https://help.okta.com/okta_help.htm?type=oie&id=ext-pam-projects

Reply:
These alias links won't work until we republish the docs again next week.